cmake/modules/BuildFIO.cmake

```diff
@@ -6,7 +6,7 @@ function(build_fio)
     DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
     UPDATE_COMMAND "" # this disables rebuild on each run
     GIT_REPOSITORY "https://github.com/axboe/fio.git"
-    GIT_TAG "fio-3.5"
+    GIT_TAG "540e235dcd276e63c57ca4bd35f70a0651e2d00e"
```

Comment:
We should come back to using fio-3.x tag once a release with your patch is out, right?

Reply:
Yes. Now just make it work.